README.md

```diff
@@ -1,4 +1,5 @@
 # zencart
+
 https://zencart.netlify.app
```

Comment:
The header is overriding the cards. Otherwise everything is fine.

Reply:
I did not get by overriding the cards